spec/operators/concatWith-spec.ts

```diff
@@ -2,14 +2,15 @@ import { expect } from 'chai';
 import { of, Observable } from 'rxjs';
 import { concatWith, mergeMap, take } from 'rxjs/operators';
 import { TestScheduler } from 'rxjs/testing';
-import { assertDeepEquals, NO_SUBS } from '../helpers/test-helper';
+import { NO_SUBS } from '../helpers/test-helper';
+import { observableMatcher } from '../helpers/observableMatcher';
 
 /** @test {concat} */
 describe('concat operator', () => {
   let rxTest: TestScheduler;
 
   beforeEach(() => {
-    rxTest = new TestScheduler(assertDeepEquals);
+    rxTest = new TestScheduler(observableMatcher);
```

Comment:
NOTE: These had to be changed, because a simple deep-equals check fails for `expect(new CustomError()).to.deep.equal(new CustomError())` once call stacks are in the mix.